python-vim-lldb/import_lldb.py

```diff
@@ -51,10 +60,12 @@ def import_lldb():
         # Unable to import lldb module from path returned by `lldb -P`
         pass
 
-    # On Mac OS X, use the try the default path to XCode lldb module
+    # On Mac OS X, try the default path to XCode lldb module
     if "darwin" in sys.platform:
-        xcode_python_path = "/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Versions/Current/Resources/Python/"
+        python_major_version = vim.eval("s:lldb_python_version")
+        xcode_python_path = "/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Versions/Current/Resources/Python%s/"% python_major_version
```

Comment:
The path for Python 2 is not `/Python2/` but just `Python`, so for Python 2 you'll want to append nothing.

Reply:
The naming of that variable is a bit misleading. In the case of Python 2 support, it resolves to an [empty string](https://github.com/67hz/vim-lldb/blob/37d4529ba447aeb562c8f6603d6d8808d06d27f3/plugin/lldb.vim#L12). A better name like `xcode_path_modifier` might make this more clear?